app/src/common/shared/org/mozilla/vrbrowser/search/suggestions/SuggestionsProvider.java

```diff
@@ -90,6 +93,11 @@ public void setComparator(Comparator comparator) {
                 items.sort(mComparator);
             }
             future.complete(items);
+
+        }).exceptionally(th -> {
+            Log.e(LOGTAG, "Error getting bookmarks suggestions: " + th.getLocalizedMessage());
```

Comment:
I assume the exceptions do not contain PII? I'm paranoid after all the work it took to clean up FxR.

Reply:
Being part of AC I guess they don't, maybe @pocmo can answer that.

Reply:
I would hope not but since these API deal with bookmarks and history I could conceive of something get leaked out.

Reply:
Since this calls into Kotlin and Rust code of AS libraries we can't really guarantee that at the AC level.

Reply:
@keianhzo we should probably make the Logs debug then so they get stripped out for release?